packages/transformers/pug/src/PugTransformer.js

```diff
@@ -4,20 +4,38 @@ import path from 'path';
 import {Transformer} from '@parcel/plugin';
 
 export default new Transformer({
-  async getConfig({asset}) {
-    const config = await asset.getConfig([
+  async loadConfig({config}) {
+    let configFile = await config.getConfig([
       '.pugrc',
       '.pugrc.js',
```

Comment:
Should we just drop the .js versions of all of the configs in this PR? I think we invented most of them right? Do other tools support these config file names? If not, then maybe we drop them until someone has a good reason for them...